### PR TITLE
Added safesearch options to startpage engine

### PR DESCRIPTION
**Pull request by gustav2302**
Added the option to turn safesearch on or off in the startpage engine

### CODE REVIEW

File: searx/engines/startpage.py [hunk 1]
```diff
@@ -114,10 +114,16 @@ def request(query, params):
     #
     # [1] https://addons.mozilla.org/en-US/firefox/addon/startpage-private-search/
 
+    #set safesearch
+    family = ''
+    if params['safesearch'] == 0:
+        family += '26edcb9d2db7168438f47a16f48020b9b73edd4a5fcaba97fb11ce8497dd03cdab5b7b508ab460c642ebc4b8477d08d78cd0d3da8251a5b51b03aada722a0f62ef8c78b2ead007b6c7a7ac'
```
Comment (gustav2302) [inline searx/engines/startpage.py:120]: This string represents the default startpage preferences but with the family filter turned off

**Comment by return42** [comment]
Hi @gustav2302, thanks a lot for your PR. The CI reported issue can be fixed by:

    make format.python

FYI: before commit its recommend to test by `make test` / we have I [Development Quickstart](https://docs.searxng.org/dev/quickstart.html) guide .. very short and worth to read ;-)

**Comment by return42** [comment]
@gustav2302 FYI the last commit adds broken code ae84fd03 / you need to solve a conflict when merge / BTW I recommend to rebase in a feature branche / do you need help, if you want I can fix the commits.

**Comment by gustav2302** [comment]
@return42 Thank you, I would really appreciate it if you fixed it. This is my first time ever making a pull request, so I'm not that experienced with git

**Comment by return42** [comment]
@gustav2302 sorry for the late response:

In PR

- https://github.com/searxng/searxng/pull/2269
 
in commit ["Revision of the locale- and language- handling in SearXNG"](https://github.com/searxng/searxng/pull/2269/commits/8ec36df3b6611142273daa7605b2ff7fc01b6750#diff-5a07bba22897433118bad961f089c47ae0446fe9fe75a0e1b95300957c82916cR286) I added time-range and save-search filter to startpage.  I'm going to close this issue .. and again: sorry for the late response :+1:

**Pull request closed without merging**